packages/storage-plus/src/de.rs

```diff
@@ -56,6 +77,11 @@ impl KeyDeserialize for String {
             // FIXME: Add and use StdError utf-8 error From helper
             .map_err(|err| StdError::generic_err(err.to_string()))
     }
+
+    #[inline(always)]
+    fn from_vec(value: Vec<u8>) -> StdResult<Self::Output> {
```

Comment:
All these cases which just dispatch to the same implementation could be handled by a default trait implementation. This would also make it clearer where we are optimising.

Note that this case is a missed chance. It calls `String::from_utf8(value.to_vec())` right after `value.as_slice()`.

`value.as_slice().to_vec()` is a hidden and unneeded clone.

Reply:
Done.